source/agora/node/Network.d

```diff
@@ -257,4 +281,22 @@ class Network
     {
         return new RestInterfaceClient!API(address);
     }
+
+    /***************************************************************************
+
+        Reads the metadata from the provided disk path.
+
+        Subclasses could override this method and return
```

Comment:
So I made something similar to `getMetadata`. The unittest code will override this function and return a `new MemMetadata` rather than `new DiskMetadata`.

Now I just need your other PR to be finished so I can add some tests on top of it. @Geod24